test/src/test/java/org/jenkins/ui/symbol/SymbolJenkinsTest.java

```diff
@@ -15,6 +15,21 @@ public class SymbolJenkinsTest {
     public RealJenkinsRule rjr = new RealJenkinsRule()
             .addPlugins("plugins/design-library.jpi", "plugins/prism-api.jpi", "plugins/bootstrap5-api.jpi");
 
+    @Test
```

Comment:
```suggestion
    @Issue("JENKINS-73243")
    @Test
```
with `import org.jvnet.hudson.test.Issue;`

Reply:
done